lib/std/os/bits/haiku.zig

```diff
@@ -383,7 +407,7 @@ pub const O_SYNC = 0x0080;
 pub const O_RSYNC = 0o4010000;
 pub const O_DIRECTORY = 0x20000;
 pub const O_NOFOLLOW = 0x0100;
-pub const O_CLOEXEC = 0x00100000;
+pub const O_CLOEXEC = 0x00000040;
```

Comment:
what was the cause of these numbers being incorrect? do all the bits need to be double-checked now?

Reply:
i believe the cause of these numbers being incorrect was a copy and paste on my part from one of the bsd ports. some numbers initially matched which probably contributed to my (false) assumption that it was basically the same. on retrospect, that was probably a hasty decision on my part .

i think it's prudent that at least the Haiku bits are double checked. i will add it to my list of things to do and perform any clean up for that.

Reply:
Thank you!